lib/modules/validator.rb

```diff
@@ -0,0 +1,19 @@
+module Validator
+  def validate_length_range?(value, length_range)
+    length_range.include?(value.length)
+  end
+
+  def validate_class?(value, klass)
+    value.is_a?(klass)
+  end
+
+  def validate_each_char_in_range?(value, first_range, last_range)
+    value.each do |num|
+      return false unless num.to_i.between?(first_range, last_range)
```

Comment:
use correct methods of array in ruby and choose the most suitable for yourself
https://ruby-doc.org/core-1.9.3/Array.html